test/e2e/sidebar.test.js

```diff
@@ -69,3 +69,96 @@ test.describe('Sidebar Tests', () => {
     expect(page.url()).toMatch(/\/test%3Efoo$/);
   });
 });
+
+test.describe('Configuration: autoHeader', () => {
+  test('autoHeader=false', async ({ page }) => {
+    const docsifyInitConfig = {
+      config: {
+        loadSidebar: '_sidebar.md',
+        autoHeader: false,
+      },
+      markdown: {
+        sidebar: `
+            - [QuickStartAutoHeader](quickstart.md)
+          `,
+      },
+      routes: {
+        '/quickstart.md': `
+            the content of quickstart space
+            ## In the main content there is no h1
+          `,
+      },
+    };
+
+    await docsifyInit(docsifyInitConfig);
+
+    await page.click('a[href="#/quickstart"]');
+    expect(page.url()).toMatch(/\/quickstart$/);
+    // const element = page.locator('#main');
```

Comment:
Any reason to keep these commented lines?

Reply:
> Any reason to keep these commented lines?

sorry for the missing clean, I will remove it.